content/en-us/reference/engine/globals/LuaGlobals.yaml

```diff
@@ -214,6 +220,12 @@ functions:
 
       `loadstring()` is disabled by default. For guidance around enabling it,
       see `Class.ServerScriptService`.
+
+      **WARNING:**  This method disables certain Luau optimizations on the returned function.
+      Extreme caution should be taken when using `Global.LuaGlobals.loadstring()`; if
+      your intention is to allow users to run code in your experience, make sure to
+      protect the returned function's environment by using `Global.LuaGlobals.getfenv()`
```

Comment:
Note that both `getfenv()` and `setfenv()` are no longer supported by Roblox, but I suppose if `loadstring()` is also so heavily cautioned against, readers will see the deprecation notes I added to `getfenv()` and `setfenv()` as well.